playground/backend/internal/setup_tools/life_cycle/life_cycle_setuper.go

```diff
@@ -143,3 +153,41 @@ func updateJavaLogConfigFile(paths fs_tool.LifeCyclePaths) error {
 	}
 	return nil
 }
+
+func prepareSbtFiles(lc *fs_tool.LifeCycle, pipelineFolder string, workingDir string) (*fs_tool.LifeCycle, error) {
+	cmd := exec.Command("sh", filepath.Join(workingDir, "new_scio_project.sh"))
```

Comment:
Can we have this as a constant too, please?

Reply:
Done

Reply:
Can't see changes here either.

Reply:
@daria-malkova could you please fix this one?

Reply:
Yep, sorry, looks like I have changed it in auto-generated file in containers...

Reply:
No worries, thanks!